cetmix_tower_server/wizards/cx_tower_plan_execute_wizard.py

```diff
@@ -5,9 +5,9 @@
 from ..models.tools import generate_random_id
 
 
-class CxTowerPlanExecuteWizard(models.TransientModel):
+class CxTowerPlanRunWizard(models.TransientModel):
     _name = "cx.tower.plan.execute.wizard"
```

Comment:
rename to "cx.tower.plan.run.wizard"

it's TransientModel

Reply:
> rename to "cx.tower.plan.run.wizard"
> 
> it's TransientModel

Although it's a transient model, its records are stored in the DB. So it raises an issue when an existing db is updated, which will require a migration script to solve.
I think we can come back to this when all other issues are fixed.